source/_components/media_player.epson.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Epson"
+description: "Instructions on how to integrate Epson projector into Home Assistant."
+date: 2018-03-24 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Media Player
+ha_release: "0.67"
+ha_iot_class: "Local Push"
+---
+
+The `epson` platform allows you to control a Epson projector from Home Assistant.
+
+To add Epson to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: epson
+    host: 192.168.0.123
+```
+
+{% configuration %}
+host:
+  description: The host name or address of the Epson projector
+  required: true
+  type: string
+port:
+  description: The HTTP port number. Defaults to 80.
+  required: false
+  type: int
+name:
+  description: The name of the device used in the frontend.
+  required: false
+  type: string
+ssl:
+  description: Enable SSL. **Feature not tested.**
+  required: false
+  type: bool
+{% endconfiguration %}
+
+Supported features of Epson projector:
+- turn on/off
+- set input
+- set/get color mode
+- increase/decrease volume via HDMI CEC
+- mute/unmute volume and video input
+- send next/previous track via HDMI CEC
```

Comment:
Since there are projectors with internal media players or speakers the command may apply to the projector itself. Thus I would remove "via HDMI CEC".